documentation/guide-usecase.asciidoc

```diff
@@ -53,23 +56,34 @@ public interface UcApproveBooking {
 ----
 
 == Implementation
-For the implementation of a use-cas, the same rules that are described for the link:guide-component-facade.asciidoc#implementation[component-facade implementation].
-
- .
-
-However, when following the use-case approach, your component facade simply changes to:
+For the implementation of a use-case, the same rules that are described for the link:guide-component-facade.asciidoc#implementation[component-facade implementation].
 
+In short, the implementation should carry its own name and the suffix `Impl` and is annotated with `@Named`. It will need access to the persistent data which is done by injecting the corresponding repository (or DAO). Furthermore, it shall not expose persistent entities from the data access layer and has to map them to transfer objects using the bean-mapper. Please refer to our bean mapping documentation for more information.
```

Comment:
We also need `@ApplicationScoped`. However, I would link to `guide-dependency-injection` here and just have a fully functional sample code so add the `@ApplicationScoped` and `@Named` to the sample impl below.